runtime/nls/port/port.nls

```diff
@@ -339,12 +339,12 @@ J9NLS_PORT_MMAP_INVALID_FILE_HANDLE.user_response=Contact your service represent
 
 # END NON-TRANSLATABLE
 
-J9NLS_PORT_LINUXDUMP_PIPE_CORE=%s setting \"%s\" specifies that the core dump is to be piped to an external program.  Attempting to rename either core or core.%d.\n
+J9NLS_PORT_LINUXDUMP_PIPE_CORE=%s setting \"%s\" specifies that the core dump is to be piped to an external program.  Attempting to rename either core or core.%d.  Review the manual for the external program to find where the core dump is written and ensure the program does not truncate it.\n
 # START NON-TRANSLATABLE
 J9NLS_PORT_LINUXDUMP_PIPE_CORE.sample_input_1=/proc/sys/kernel/core_pattern
-J9NLS_PORT_LINUXDUMP_PIPE_CORE.sample_input_2=|/usr/libexec/abrt-hook-ccpp /var/spool/abrt %p %s %u %c
+J9NLS_PORT_LINUXDUMP_PIPE_CORE.sample_input_2=|/usr/lib/systemd/systemd-coredump %P %u %g %s %t %c %h
 J9NLS_PORT_LINUXDUMP_PIPE_CORE.sample_input_3=12345
-J9NLS_PORT_LINUXDUMP_PIPE_CORE.explanation=Tools such as ABRT use /proc/sys/kernel/core_pattern to redirect the core file to an external program, for example to create bug reports.
+J9NLS_PORT_LINUXDUMP_PIPE_CORE.explanation=Tools such as systemd-coredump use /proc/sys/kernel/core_pattern to process the core dump, for example to send it to a different directory and manage total core dump disk usage, or to suppress it.
```

Comment:
This is the other way around: `core_pattern` is how you configure a system to use `systemd-coredump`; I don't think the tool consults that file (it only considers the command-line arguments it gets as specified in `core_pattern`).

Reply:
Good point. How about this?

> /proc/sys/kernel/core_pattern starting with | is used to redirect the core file to tools such as systemd-coredump which processes the core dump, for example to send it to a different directory and manage total core dump disk usage, or to suppress it.